### DISCUSSION TITLE
Adding to an existing next.js project?

### DISCUSSION THREAD

**Discussion by onlyphantom** [daltonmenezes/opendocs#6] [Ideas]
Very cool project, thanks for sharing with the world your work on this! 

Would be nice to have a guide on how this can be incorporated into an existing next.js app though. Would probably be something of the format of :

1. `yarn add __ __ __` (or `pnpm`) if the existing app don't yet have these dependencies
2. create a sub directory under `/app` (or whatever makes sense), and update the config file accordingly to point to this path
3. start adding `.mdx` files 
4. link it to the build command by doing _, _

Would greatly help folks with existing nextjs apps wanting to give it a try! Also happy to contribute to testing / reporting bugs etc, let me know!

**Comment by daltonmenezes** [comment]
Hi @onlyphantom , thanks!

It's something to think about. At the moment, I think that the opposite path would be better: migrating the existing project to opendocs (since opendocs is a boilerplate) and not the other way around, since the person has mastery of their current project, it would be enough to just update the app/page.tsx and app/layout.tsx part from opendocs to change the site, since the blog and docs are ready and maybe that's what they don't have yet. I say this because we need to think about the simplest path, but nothing prevents everyone from contributing to adding migration documents to opendocs for both cases. But the case I mentioned would make it simpler to follow, what do you think?
